src/Compilers/VisualBasic/Test/Emit/Attributes/InternalsVisibleToAndStrongNameTests.vb

```diff
@@ -1653,6 +1653,21 @@ End Class
     End Sub
 
     <Fact>
+    Public Sub PublicSign_NoKey()
```

Comment:
When I wrote this, I had something in mind, but I can't remember now. I'll see what I was thinking.